ivy/container/linear_algebra.py

```diff
@@ -1763,6 +1763,57 @@ def static_outer(
         map_sequences: bool = False,
         out: Optional[ivy.Container] = None,
     ) -> ivy.Container:
+        """
+            ivy.Container static method variant of ivy.outer. This method simply wraps the
+            function, and so the docstring for ivy.outer also applies to this method with
+            minimal changes.
+
+            Computes the outer product of two arrays, x1 and x2, by computing the tensor
+            product along the last dimension of both arrays.
+
+            Parameters
+            ----------
+            x1
+                first input array having shape (..., N1)
+            x2
+                second input array having shape (..., N2)
+            key_chains
+                The key-chains to apply or not apply the method to. Default is ``None``.
+            to_apply
+                If True, the method will be applied to key_chains, otherwise key_chains will be
+                skipped. Default is ``True``.
+            prune_unapplied
+                Whether to prune key_chains for which the function was not applied.
+                Default is ``False``.
+            map_sequences
+                Whether to also map method to sequences (lists, tuples).
+                Default is ``False``.
+            out
+                optional output container, for writing the result to. The container must have
+                shape (..., N1, N2). The first x1.ndim-1 dimensions must have the same size as
+                those of the input array x1 and the first x2.ndim-1 dimensions must have the same
+                size as those of the input array x2.
+
+            Returns
+            -------
+            ret
+                an ivy container whose shape is (..., N1, N2). The first x1.ndim-1 dimensions
+                have the same size as those of the input array x1 and the first x2.ndim-1
+                dimensions have the same size as those of the input array x2.
+
+            Example
+            -------
+            >>> x1 = ivy.array([[1, 2, 3], [4, 5, 6]])
+            >>> x2 = ivy.array([1, 2, 3])
+            >>> y = ivy.static_outer(x1, x2)
```

Comment:
Can you use container inputs for this example.
and you should call the function by doing `ivy.Container.static_outer(x1, x2)`

Reply:
Done Kindly Review Again